.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.5.0
+    rev: v5.0.0
```

Comment:
What is this one?

Reply:
Hooks which we consume from the `pre-commit` project - they are not shipped automatically so we specify them like that.